sdb/commands/filter.py

```diff
@@ -17,7 +17,7 @@
 # pylint: disable=missing-docstring
 
 import argparse
-from typing import Iterable
+from typing import Iterable, List, Optional
 
```

Comment:
Let's update the `filter` doc-string to explain the requirements around quoting expressions. Are the examples in the current string still valid?

Reply:
Added a follow-up commit that the user is not forced to use quotes  around filter and all the examples are updated in the docstring (I was planning to do this as a follow-up PR but why not now :) )